internal/core/cobra_builder.go

```diff
@@ -114,10 +112,10 @@ func (b *cobraBuilder) hydrateCobra(cobraCmd *cobra.Command, cmd *Command) {
 		cobraCmd.Annotations["SeeAlsos"] = cmd.seeAlsosAsStr()
 	}
 
-	cobraCmd.PreRunE = cobraPreRunInitMeta(b.ctx, cmd)
+	cobraCmd.PreRunE = cobraPreRunInitMeta(newMetaContext(b.meta), cmd)
```

Comment:
We don't want to use the same context for `PreRunE` and `Run`?
Maybe we should create a `ctx` variable above and use it for `PreRunE` and `Run`.